chromadb/test/property/strategies.py

```diff
@@ -158,12 +204,21 @@ def collections(draw, add_filterable_data=False, with_hnsw_params=False):
             key = draw(safe_text)
             known_metadata_keys[key] = draw(st.sampled_from(safe_values))
 
-    has_documents = draw(st.booleans())
+    if has_documents is None:
+        has_documents = draw(st.booleans())
     if has_documents and add_filterable_data:
         known_document_keywords = draw(st.lists(safe_text, min_size=5, max_size=5))
     else:
         known_document_keywords = []
 
+    if not has_documents:
+        has_embeddings = True
```

Comment:
As opposed to supporting the F/F case and casting it to true, can we top-level assert that its NOT allowed so that the test documents our assumptions